provisioning/templates-api/kubernetes/templates/etcd-values.yaml

```diff
@@ -28,7 +28,7 @@ auth:
     create: true
     allowNoneAuthentication: false
   token:
-    type: simple
+    type: jwt
```

Comment:
Bez Helm-u sme nevedeli nasetupovat defaultne `jwt` tokeny (ktore si etcd generuje interne).,
Teraz to uz ide, Helm vygeneruje private key.

Je to rychlejsi a bezpecnejsi sposob, ako `simple`.
Je to aj default hodnota: 
https://github.com/bitnami/charts/blob/638a48e4d3ec7b5d160f4b525ec40218512c464b/bitnami/etcd/values.yaml#L119-L131

![image](https://user-images.githubusercontent.com/19371734/199055498-33fbe789-2c48-42c9-aecd-9b103a09d94c.png)


------

Novy secret:

![image](https://user-images.githubusercontent.com/19371734/199055040-8613bb42-d348-4b4f-9b2a-fd4143c76d48.png)

Reply:
Rozumim tomu spravne tak ze pro prvotni autentizaci se pouzije jmeno/heslo https://github.com/keboola/keboola-as-code/blob/75923882f959e6f06353e4c0153be6be1507bd6a/provisioning/templates-api/kubernetes/templates/templates-api.yaml#L51 ale pak uz se autentizuje [jwt tokenem](https://etcd.io/docs/v3.5/learning/design-auth-v3/#authentication)?